homeassistant/components/webostv/media_player.py

```diff
@@ -241,6 +241,18 @@ def _update_states(self) -> None:
             name=self._device_name,
         )
 
+        if (
+            self._client.media_state is not None
+            and self._client.media_state.get("foregroundAppInfo") is not None
+        ):
```

Comment:
We set the `self._attr_assumed_state = True` in https://github.com/home-assistant/core/blob/3a8494cb88c1df720f0a12403254d236baa460c0/homeassistant/components/webostv/media_player.py#L124 which cause the frontend to display seperate Play & Stop buttons since we don't know the play state, when you know it, we should set it to `False` so the frontend will display the correct Play/Pause state.
```suggestion
        if (
            self._client.media_state is not None
            and self._client.media_state.get("foregroundAppInfo") is not None
            self._attr_assumed_state = False
        ):
```
See https://github.com/home-assistant/core/pull/66486 for more information about it

Reply:
It's supported for every streaming and media player app that I've tried (youtube, netflix, amazon, spotify, built in media player app, jellyfin), so I assume it is for all.